commcare_connect/audit/tests/factories.py

```
@@ -0,0 +1,26 @@
+    opportunity = factory.SubFactory(OpportunityFactory)
+    period_start = datetime.date(2026, 4, 13)
+    period_end = datetime.date(2026, 4, 19)
+
+
+class AuditReportEntryFactory(DjangoModelFactory):
+    class Meta:
+        model = AuditReportEntry
+
+    audit_report = factory.SubFactory(AuditReportFactory)
+    opportunity_access = factory.SubFactory(OpportunityAccessFactory)
+    results = {}
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)